ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1578,6 +1578,35 @@ literals:
 - valve_status:
   - OPEN
   - CLOSED
+
+- failed_panel_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- medium_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- particle_concentration_alarm:
```

Comment:
This is not specific enough, please look at existing fields like `high_zone_co2_concentration_alarm` for examples.

Reply:
i see. I'll ask how this point is used as it could be high or low alarm and if this point is programmable (can either be set as high or low).

Reply:
`high_zone_air_h2_concentration_alarm` i guess would make it specific. Thanks